src/common_symbols2.inc

```diff
@@ -127,7 +127,6 @@ jl_symbol("pipe_writer"),
 jl_symbol("idxfloor"),
 jl_symbol("id"),
 jl_symbol("ComplexF32"),
-jl_symbol("/home/jeff/src/julia/usr/share/julia/site/v0.7/Pkg/src/resolve/versionweight.jl"),
```

Comment:
I do use this symbol all the time.

Reply:
The data we have says that you do, in fact, use it all the time :)